docs/learning/Display-Comparison.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Display Comparison
+nav_exclude: false
+has_children: false
+parent: Learning
+search_exclude: false
+last_modified_date: 2023-02-04
+---
+
+# Display Comparison
+
+## Why is this important?
+Displays are not exactly the same, and different display types have different pros and cons.
+
+## Panel Types:
+
+### Twisted Nematic (TN)
+they are the most common type of LCD panels that is still used and manufactuered today. They offer fast response times and high refresh rates, yet they have limited viewing angles and poor colors.
+
+### Vertical Alignment (VA)
+Has more saturated colors and higher contrast ratio that TN panels, making them a good choice for watching movies and other multimedia content. But they have the slowest responce times between the 3 types of panels.
+
+### In-Plane Switching (IPS)
+IPS is the newset technology between the 3. They offer the best colors reproduction, having a wide viewing angels and relatively low response times. However they are more expensive than the other two and can suffer from backlight bleeding defect. 
+
+### Main similarities between TVs and monitors:
+Both TV and monitor are electronic display devices that show images and videos, they share common video input ports like HDMI and VGA. They also came with same panel types such as IPS, TN, and VA.
```

Comment:
Both TVs and monitors are electronic display devices that show images and videos. They share common video input ports like HDMI and VGA. They also come with the same panel types such as IPS, TN, and VA.